src/components/EventDetails.js

```diff
@@ -1,27 +1,28 @@
 import React from 'react';
 import { Button } from './Button';
 
-export const EventDetails = (event) => {
+export const EventDetails = (props) => {
   return(
     <div className='event-details-card'>
       <div className="event-details-container">
-        <h3>{event.name.text}</h3>
-        <img alt="Event" src={event.logo.url} />
+        <h3>{props.event.name.text}</h3>
+        <img alt="Event" src={props.event.logo.url} />
```

Comment:
Is it necessary to add a check like ```props.event.logo ? <img src={props.event.logo.url}/> : null``` in here? I'm not sure either but wanted to point it out. While I was trying to get event images inside the eventsByCategoryPage component, I was getting error without this check